app/app.go

```diff
@@ -458,13 +476,15 @@ func New(
 		wasmOpts...,
 	)
 
+	ibcRouter := porttypes.NewRouter()
+
 	// register wasm gov proposal types
-	enabledProposals := GetEnabledProposals()
+	// The gov proposal types can be individually enabled
 	if len(enabledProposals) != 0 {
```

Comment:
When `enabledProposals` hasn't been set here doesn't this make the whole block redundant?
Why do we not want to check proposals using the same pattern as before? Perhaps I'm missing something.

Reply:
@the-frey I modeled this after the style used in wasmd.  These changes bring us much closer to being a wasmd reference implementation.